clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SearchController.groovy

```diff
@@ -45,15 +48,28 @@ class SearchController {
    */
   @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission()")
   @RequestMapping(value = '/search', method = RequestMethod.GET)
-  List<SearchResultSet> search(SearchQueryCommand q) {
+  List<SearchResultSet> search(@RequestParam(value = "q", defaultValue = "", required = false) String query,
+                               @RequestParam(value = "type") List<String> type,
+                               @RequestParam(value = "platform", required = false) String platform,
+                               @RequestParam(value = "pageSize", defaultValue = "10000", required = false) int pageSize,
+                               @RequestParam(value = "page", defaultValue = "1", required = false) int page,
+                               HttpServletRequest httpServletRequest) {
+
+    def filters = httpServletRequest.getParameterNames().findAll { String parameterName ->
```

Comment:
Looking at the EntityTagsController, we're doing something similar that's a little cleaner than pulling things directly off the HttpServletRequest. You could do something like this:
```groovy
   ...
   @RequestParam(value = "page", defaultValue = "1", required = false) int page,
   @RequestParam Map<String, Object> filters) {
["q", "type", "platform", "pageSize", "page"].each { filters.remove(it) }
```
This is a little cleaner and easier to read/understand IMO